app.example.env

```diff
@@ -1,13 +1,36 @@
-DB_HOST= # host name of the database (for example: localhost)
-DB_PORT= # port of the database (for example: 3306)
-DB_NAME= # name of the database
-DB_USER= # username of the database (for example: root)
-DB_PASSWORD= # password of the database
-
-AZURE_CLIENT_ID= # client id of the azure application service principal
-AZURE_CLIENT_SECRET= # client secret of the azure application service principal
-AZURE_TENANT_ID= # tenant id of the azure application service principal
-AZURE_STORAGE_ACCOUNT_NAME= # name of the azure storage account
-AZURE_STORAGE_BLOB_CONTAINER_NAME= # name of the blob storage container
-
-PORT= # port of the application (optional)
\ No newline at end of file
+# Database Configuration
+DB_HOST= # Hostname of the database (e.g., localhost)
+DB_PORT= # Port of the database (e.g., 5432 for PostgreSQL, 3306 for MySQL)
+DB_NAME= # Name of the database
+DB_USER= # Username for the database (e.g., root)
+DB_PASSWORD= # Password for the database
+
+# Azure Configuration
+AZURE_CLIENT_ID= # Client ID of the Azure application service principal
+AZURE_CLIENT_SECRET= # Client Secret of the Azure application service principal
+AZURE_TENANT_ID= # Tenant ID of the Azure application service principal
+AZURE_STORAGE_ACCOUNT_NAME= # Name of the Azure Storage account
+AZURE_STORAGE_BLOB_CONTAINER_NAME= # Name of the Azure Blob Storage container
+
+# Server Configuration
+PORT= # Port for the application to run on (optional, default is 8080)
+
+# JWT Configuration
+JWT_SECRET= # Secret key used for signing JWT tokens
+JWT_LIFESPAN= # Lifespan of the JWT token in milliseconds (e.g., 86400000 for 24 hours)
+JWT_ALGORITHM= # Algorithm used for signing JWT tokens (e.g., HS256)
+JWT_VERIFICATION_DURATION= # Duration for which a verification token is valid, in milliseconds
```

Comment:
Verification duration is not **strongly related** to JWT

Reply:
Ok